modules/Bio/EnsEMBL/Production/Pipeline/GTF/DumpFile.pm

```diff
@@ -70,39 +70,17 @@ sub param_defaults {
 
 sub fetch_input {
   my ($self) = @_;
-  
-  my $eg = $self->param('eg');
-  $self->param('eg', $eg);
-
-  if($eg){
-     my $base_path  = $self->build_base_directory();
-     $self->param('base_path', $base_path);
- 
-     my $release = $self->param('eg_version');    
-     $self->param('release', $release);
-  } 
-  
+
   throw "Need a species" unless $self->param('species');
   throw "Need a release" unless $self->param('release');
   throw "Need a base_path" unless $self->param('base_path');
 
-  if(!$eg){
-   throw "No gtfToGenePred executable given" 
```

Comment:
Do we not still need to check for these files? Seems like they are used in vertebrate context.

Reply:
@james-monkeyshines: Thanks for spotting this. I think I was a bit overzealous when cleaning up. I think it's worth checking that these exists and are executable for vertebrates. I've pushed a new change.